templates/experiments/experiment_create.html

```diff
@@ -0,0 +1,14 @@
+{% extends 'web/app/app_base.html' %}
+{% load form_tags %}
```

Comment:
For my curiosity, where does `form_tags` come from?

Reply:
`apps/web/templatetags/form_tags.py` - it's a Pegasus thing.

I'm considering trying out It would be nice to use crispy forms but

Reply:
Ah there it is, thanks!

> I'm considering trying out https://github.com/django-crispy-forms/crispy-tailwind

I have no reservations against it